docs/migrating_to_nextcord.rst

```diff
@@ -8,3 +8,46 @@ Migrating to nextcord
 Due to the `original discord.py repository <https://github.com/Rapptz/discord.py>`_ becoming read-only, we decided
 that it would be necessary to fork it and keep on developing further. We also wanted to change the name and voted on
 nextcord in order to properly register it at pypi.
+
+Porting from discord.py
+-------------------------
+
+In order to port a bot using discord.py to nextcord.py, follow these steps:
+
+1. Install nextcord: 
+
+.. code:: sh
+
+    # Linux/macOS
+    python3 -m pip install -U nextcord
+
+    # Windows
+    py -3 -m pip install -U nextcord
+
+2. Update the following import statements:

```

Comment:
This is not neccessary (although it is recommended). 

Maybe say that we recommend doing it although it will probably still work.
Maybe link to the migrating docs here instead of the code example?